transforms/ember-object/__testfixtures__/app/users/default-import.export.output.js

```diff
@@ -0,0 +1 @@
+export default class UsersDefaultImportExportInputRoute extends Route {}
```

Comment:
This test does test whether or not the folder and the file name will be prepended to the class, but I think we should explicitly test the case where the file name _is_ the name of the class/type (e.g. `route.js`). This is definitely going to be the most common case with pods, and I'd like to make sure that it's under test.

Reply:
Is it possible to setup that kind of test, seems output and input need to be used?

Reply:
@pzuraq any thoughts on how we'd set this up?

Reply:
Just recording here real quick because I'm not sure when @rwjblue  or I will have time do it, but we chatted and figured the best solution here would be to rename the input files so that they drop the `.input` part before running tests in codemod-cli